packages/domains-table/src/domains-table/domains-table-ssl-cell.tsx

```diff
@@ -3,6 +3,7 @@ import { Icon } from '@wordpress/components';
 import { lock } from '@wordpress/icons';
 import clsx from 'clsx';
 import { useTranslate } from 'i18n-calypso';
+import { navigate } from 'calypso/lib/navigate'; // eslint-disable-line no-restricted-imports
```

Comment:
```suggestion
import page from '@automattic/calypso-router';
```

Can we import this library instead? That'll remove the need to use the eslint escaping

Reply:
Thanks for the suggestion @Imran92! Works perfectly.

Reply:
It looks like it was the cause of failing tests :-/
Though, it didn't show up in test logs.